tests/operators/test_density_for_op.py

```diff
@@ -27,8 +28,14 @@ def radial_distance(traj, shape):
 @parametrize(backend=["gpunufft", "tensorflow"])
 def test_pipe(backend, traj, shape, osf):
     """Test the pipe method."""
+    if backend == "tensorflow" and osf == 1:
+        result = pipe(traj, shape, backend, num_iterations=10)
+    elif backend == "tensorflow" and osf != 1:
+        pytest.skip("Tensorflow does not support OSF != 1")
+    else:
+        result = pipe(traj, shape, backend, osf=osf, num_iterations=10)
```

Comment:
```suggestion
    if backend == "tensorflow" and osf != 1:
        pytest.skip("Tensorflow does not support OSF != 1")
    result = pipe(traj, shape, backend, osf=osf, num_iterations=10)
```

Reply:
just simplifying a bit.

Reply:
i thought of doing this but tensorflow mri doesn't take osf arguments: so to avoid having an unused argument in pipe i added conditions in test

Reply:
we can add this in the CI , i test on my machine and it work

Reply:
> 

i thought of doing this but tensorflow mri doesn't take osf arguments: so to avoid having an unused argument in pipe i added conditions in test

In this case, I would recommend having osf=1 as a dummy argument in tfnufft pipe (and raise an error there). This would make a consistent API call for all pipe methods.

Reply:
By default osf=2 internally within `tensorflow-mri` I recommend using the same default value.